python/ray/data/read_api.py

```diff
@@ -4421,6 +4422,118 @@ def read_delta(
     )
 
 
+@PublicAPI(stability="alpha")
+def read_kafka(
+    topics: Union[str, List[str]],
+    *,
+    bootstrap_servers: Union[str, List[str]],
+    trigger: Literal["once"] = "once",
+    start_offset: Optional[Union[int, str]] = "earliest",
+    end_offset: Optional[Union[int, str]] = "latest",
+    authentication: Optional[Dict[str, Any]] = None,
```

Comment:
What happens if this is `None`? If it's implicitly required, I think it'd be clearer if this was a required argument.